File: src/uri_beecrowd/Accepted/ADHOC/P2227_Airport.py
# Date: 08-04-2020
# author : Teerapat Phokhonwong
# Online judge: URI Online Judge
# Categories: MATHEMATICS
# Problem: 2227 - Airport
# Link: https://www.urionlinejudge.com.br/judge/en/problems/view/2227
# Timelimit: 1 sec
# Status: Accepted
# Submission: 4/8/20, 1:16:11 PM
# Language: Python3
# Runtime: 0.316s
# Solution:
# Note:

if __name__ == "__main__":
    t = 0
    while True:
        a, v = map(int, input().split())
        if a == v == 0:
            break
        t += 1
        print('Teste', t)
        airport = [0] * (a + 1)
        most_traffic = 0
        for i in range(v):
            x, y = map(int, input().split())
            airport[x] += 1
            airport[y] += 1
            if most_traffic < airport[x]:
                most_traffic = airport[x]
            if most_traffic < airport[y]:
                most_traffic = airport[y]

        isPrinted = 0
        for i in range(1, a + 1):
            if airport[i] == most_traffic:
                print(str(i) + ' ', end='')
        print('\n')

File: Java/URI_BEECROWD/AD_HOC/P2227_Airport.java
/**
 * @author Teerapat Phokhonwong
 * @Onlinejudge: URI ONLINE JUDGE
 * @Categories: AD-HOC
 * @Problem: 2227 - Airport
 * @Link: https://www.urionlinejudge.com.br/judge/en/problems/view/2227
 * @Timelimit: 1 sec
 * @Status: Accepted
 * @Submission: 4/8/20, 2:12:41 PM
 * @Runtime: 0.260s
 * @Solution:
 * @Note:
 */


package URI.Accepted.AD_HOC.sourcecode;

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.BufferedWriter;
import java.io.OutputStreamWriter;
import java.io.IOException;


public class P2227_Airport {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        String line = "";
        int t = 0;
        while (!(line = br.readLine()).equals("0 0")) {
            String[] st = line.split(" ");
            int a = Integer.parseInt(st[0]);
            int v = Integer.parseInt(st[1]);
            int[] airport = new int[a + 1];
            int most_traffic = 0;
            for (int i = 0; i < v; i++) {
                st = br.readLine().split(" ");
                int x = Integer.parseInt(st[0]);
                int y = Integer.parseInt(st[1]);
                airport[x]++;
                airport[y]++;
                if (most_traffic < airport[x]) most_traffic = airport[x];
                if (most_traffic < airport[y]) most_traffic = airport[y];
            }
            bw.append("Teste " + (++t) + "\n");
            for (int i = 1; i <= a; i++) {
                if (airport[i] == most_traffic) {
                    bw.append(i + " ");
                }
            }
            bw.append("\n\n");
        }
        bw.flush();
    }

}
